README.md

```diff
@@ -36,8 +36,10 @@ Features
 ========
 
 uftrace traces each function in the executable and shows time durations.
+
+With fully dynamic dynamic tracing (`-la`), it works on unmodified executables:
```

Comment:
Double dynamic? :)  Also the option should be `-P`.

Reply:
Thanks, I've fixed and updated the paragraph now by adding more explations:

> uftrace traces each function in the executable and shows time durations.
>
> Usually, for this to be possible, the program needs to be compiled with
`-pg` or `-fpatchable-function-entry=2`, but with fully dynamic tracing
(`-P.`|`--patch=.`), uftrace works on all executables (as long they are
not stripped, or symbol information is available from a separate file).
>
>uftrace is able to perform nested library call tracing even on stripped
executables with no symbol information using (`-l`|`--nest-libcall`)
by hooking into the procedure linkage tables of shared libraries.
>
>Using (`-a`|`--auto-args`), uftrace automatically records arguments and
return values of known functions. Without extra debug information, this
includes the API functions of standard (C language or system) libraries.
>
>If debug information for the program (`gcc -g`) is available, `--auto-args`
works even on functions inside of the compiled the user programs.
>
>In case argument information is not available, argument specifiations like
(`-A udev_new@arg1/s`) can be passed on the command line or an options file.

Reply:
Actually the dynamic tracing option `-P` takes a pattern (usually in regex, but you can use wildcard with additional `--match=glob` option) to match function names.  If the pattern doesn't contains any special character in regex, it'd try the simple full-string match.  If it contains any special character then it'd try the regex match.  That's why we use the single dot (`.`) mostly to match every function in the simplest way (with regex).

Reply:
Thanks, I submitted https://github.com/namhyung/uftrace/pull/1664 to add this helpful information to doc/uftrace-live.md and doc/uftrace-record.md. It needed elsewhere as well, I propose to follow it ip in #1664.